src/components/Person/TablePerson.tsx

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  person: Person,
+  selectedPersonSlug: string,
+  mother: Person | null,
+  father: Person | null,
+};
+
+export const TablePerson: React.FC<Props> = ({
+  person,
+  selectedPersonSlug,
+  mother,
+  father,
+}) => {
+  const isSelected = person.slug === selectedPersonSlug;
+
+  const personMather
+  = mother
+    ? <PersonLink person={mother} />
+    : person.motherName || '-';
+
+  const personFather
+       = father
+         ? <PersonLink person={father} />
+         : person.fatherName || '-';
```

Comment:
ternary operator should return one value depending on condition, don't make another condition in the ternary operator